### PR TITLE
Move Ionic sample code into new folder hierarchy

### PR DESCRIPTION
**Pull request by patniemeyer**
This moves Sam's ionic code into the new folder hierarchy.

### CODE REVIEW

File: client/ionic/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (advayDev1) [inline client/ionic/.gitignore]: We have multiple gitignores - just use one?

Comment (advayDev1) [reply client/ionic/.gitignore]: nvm, it appears this structure relies on it.

Comment (patniemeyer) [reply client/ionic/.gitignore]: They can get pretty hairy and specific to the sub-projects. It's sometimes a little cleaner to target them where needed.  But perhaps Sam intends to merge them.

File: client/ionic/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            android:grantUriPermissions="true">
+            <meta-data
+                android:name="android.support.FILE_PROVIDER_PATHS"
+                android:resource="@xml/file_paths"></meta-data>
+        </provider>
+    </application>
+
+    <!-- Permissions -->
+
+    <uses-permission android:name="android.permission.INTERNET" />
```
Comment (advayDev1) [inline client/ionic/android/app/src/main/AndroidManifest.xml]: A TODO and a linked issue to review all of these permissions.

**Comment by hspinks** [comment]
Structure looks good to me. Do we want to update the Github Action to run the Ionic tests instead of the React Native prototype tests?

File: client/ionic/android/app/src/main/assets/capacitor.config.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "appId": "io.ionic.starter",
```
Comment (advayDev1) [inline client/ionic/android/app/src/main/assets/capacitor.config.json]: Should not be using ionic's namespace

File: client/ionic/android/app/src/main/res/xml/config.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='utf-8'?>
+<widget version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+  <access origin="*" />
```
Comment (advayDev1) [inline client/ionic/android/app/src/main/res/xml/config.xml]: A TODO and issue to review this.

**Comment by advayDev1** [comment]
General
- [ ] Please add you and Sam to LICENSE list of authors
- [ ] Merge blocks on #50 for licenses
- [ ] Please use a meaningful PR name since it will appear in the merge commit.

File: client/ionic/android/app/src/main/assets/capacitor.config.json (hunk 3 above)
Comment (patniemeyer) [reply client/ionic/android/app/src/main/assets/capacitor.config.json]: I have updated these to `int.who.app`.

File: client/ionic/android/app/src/main/res/xml/config.xml (hunk 4 above)
Comment (patniemeyer) [reply client/ionic/android/app/src/main/res/xml/config.xml]: https://github.com/WorldHealthOrganization/app/issues/55

File: client/ionic/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (patniemeyer) [reply client/ionic/android/app/src/main/AndroidManifest.xml]: Created issue: https://github.com/WorldHealthOrganization/app/issues/56

Comment (myire) [reply client/ionic/android/app/src/main/AndroidManifest.xml]: todo?

File: client/ionic/android/app/src/main/res/xml/config.xml (hunk 4 above)
Comment (myire) [reply client/ionic/android/app/src/main/res/xml/config.xml]: TODO?

File: client/ionic/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (patniemeyer) [reply client/ionic/android/app/src/main/AndroidManifest.xml]: (Yes, added in 57e153f, just failed to mention it)

File: client/ionic/android/app/src/main/res/xml/config.xml (hunk 4 above)
Comment (patniemeyer) [reply client/ionic/android/app/src/main/res/xml/config.xml]: (Yes, added in 57e153f, just failed to mention it)

**Comment by advayDev1** [comment]
LGTM.

File: client/ionic/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (myire) [reply client/ionic/android/app/src/main/AndroidManifest.xml]: sorry didnt see it!

**Comment by myire** [comment]
LGTM

**Pull request merged**